strongbox-storage/strongbox-storage-layout-providers/strongbox-storage-maven-layout/strongbox-storage-maven-layout-provider/src/test/java/org/carlspring/strongbox/testing/artifact/MavenTestArtifact.java

```diff
@@ -65,4 +66,9 @@
      */
     String packaging() default "jar";
 
+    /**
+     * Additional artifact size in bytes.
+     */
+    @AliasFor(annotation = TestArtifact.class)
+    long size() default 1024;
```

Comment:
I think this should default to `0` and there should be a check for this in the generator. (`1024` is an most-likely not enough in order to create a proper jar with a manifest and all that).

Reply:
This size is size of additional file in artifact - it's not size of whole artifact. After adding files and pom to archive checksum will be generated and size of whole jar will be bigger than given size. It's more like minimal size of artifact and at the end it should be bigger than given size.

Reply:
Yes, thanks for reminding me of this. The size is the one of the random binary file that's included inside the `jar`.

Reply:
How much then? 1000000?

Reply:
Yeah, I think that'll be better.